src/main/java/com/jcraft/jsch/AgentConnector.java

```diff
@@ -1,36 +1,36 @@
 /* -*-mode:java; c-basic-offset:2; indent-tabs-mode:nil -*- */
```

Comment:
I think we may want to also remove all these editor hints at the top of all the files too?

Reply:
yes makes sense

Reply:
Did we want to handle removal of these editor hints in this PR or do that separately

Reply:
I will add it to this one